### ISSUE TITLE
3D model black on certain Android devices since v1.9.0 

### ISSUE THREAD

**Issue by alansegar** [google/model-viewer#3466]
### Description

Astronaut model on modelviewer.dev and different models on other sites are completely black on certain Android devices when upgrading beyond v1.8.0. I.e. it works OK on 1.8.0 or lower but fails on 1.9.0 or higher on the same device / browser.
Console shows error: `model-viewer.js:16340 THREE.WebGLRenderer: EXT_color_buffer_float extension not supported.`
`model-viewer.js:16340 `

```
		init: function ( capabilities ) {

			if ( capabilities.isWebGL2 ) {
 // ****fails here****
				getExtension( 'EXT_color_buffer_float' );
			} else {

				getExtension( 'WEBGL_depth_texture' );
				getExtension( 'OES_texture_float' );
				getExtension( 'OES_texture_half_float' );
				getExtension( 'OES_texture_half_float_linear' );
				getExtension( 'OES_standard_derivatives' );
				getExtension( 'OES_element_index_uint' );
				getExtension( 'OES_vertex_array_object' );
				getExtension( 'ANGLE_instanced_arrays' );

			}

			getExtension( 'OES_texture_float_linear' );
			getExtension( 'EXT_color_buffer_half_float' );
			getExtension( 'WEBGL_multisampled_render_to_texture' );

		},

		get: function ( name ) {

			const extension = getExtension( name );

			if ( extension === null ) {
// This is output to console from line getExtension( 'EXT_color_buffer_float' );
				console.warn( 'THREE.WebGLRenderer: ' + name + ' extension not supported.' ); 

			}

			return extension;

		}

	};
```
#### Live Demo
Can be seen on modelviewer.dev with the version deployed on there today

### Version
- model-viewer: v1.9.0 onwards

### Browser Affected

- [x] Chrome, version: 100.0.4896.127
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR
N/A

**Comment by elalish** [comment]
Yes, this logic changed and went back and forth a bit (have you tested all the versions beyond 1.9?). We upgraded to floating point textures internally for PMREM, which simplified the code and improved the speed quite a bit. We couldn't find any devices that didn't support the extension we needed, but it was pretty likely something existed somewhere. What device(s) have you found this problem on?

**Comment by alansegar** [comment]
I've just tested on 1.11.0 and 1.12.0. It behaves the same as 1.9.0 on both of those. The only device I know of currently with this issue is Huawei P8 (quite an old device).

**Comment by elalish** [comment]
Well, that's good at least. It's just not possible for us to support every old device, though we make a best effort especially for the more common ones. At least the old library versions are always available if you really need it for that device.
